### ISSUE TITLE
scrutiny

### ISSUE THREAD

**Issue by typophyllum** [SpeciesFileGroup/taxonworks#2063]
In OSF / SFS it is possible to add scrutinies to the taxon pages. They consist of author, year, and text (text is optional, but practically always present in OSF). The text usually points out or explains nomenclatural issues related to the corresponding taxon. It is possible to search for these scrutinies by author, year, and words used in the text.

How do I add this information appropriately in TW?

**Comment by mjy** [comment]
@typophyllum in TaxonWorks we have both more specific and generalized mechanisms for annotating data quality.

Our general corresponding term is the "Confidence".  You can create confidence levels as needed, with specific definitions, and tag records with them.  We will ultimately add facets to let you search by who created the Confidence, and when (these data are already captured for all data in TaxonWorks, we call this "Housekeeping".  I would recommend this approach to start as it puts you on the path to being able to share the data to others such that they will be able to understand them according to other standards, i.e. anticipate ultimately matching confidence levels to schemas/ontologies etc. others use.

If you need richer annotations then there are other options, but these wont' be assumed to be Confidence/Scrutiny by TaxonWorks, they will just be other attributes that can be displayed and searched on.  For example you can create Predicates and use them with DataAttributes, providign whatever text you want, or Notes.  

I suspect, but am not confident, that @LocoDelAssembly and @proceps added Scrutinies as DataAttributes, the Predicate being something like (guessing here) "SF:Scrutiny".

**Comment by LocoDelAssembly** [comment]
> I suspect, but am not confident, that @LocoDelAssembly and @proceps added Scrutinies as DataAttributes, the Predicate being something like (guessing here) "SF:Scrutiny".

It is added as an internal attribute in a debug-like string format at present. We discussed ways to improve this mapping and considered adding it as content with topic "Scrutiny" and use attribution to place year and authors there but still I suggested to open this issue to check for better alternatives.

The Scrutiny form in SFS is like this:
![image](https://user-images.githubusercontent.com/4991925/109318375-7dd5f480-782c-11eb-9249-905f8da6ff1e.png)
(Authors is people instances, not a text field)

**Comment by mjy** [comment]
The problem with this format is it evolves to be a note.  I looked at some of the existing values.  Some are expressing confidence.  Some are expressing potential issues with the assertions.  Some are just random notes that don't appear to be anything related to a Scrutiny.

Author, Year = A source of Type Person.  This is well defined.  Linking that is Citation.  IIRC Data Attributes can be Cited, we can also make Confidences citable.  

Spoke with @jlpereira.  We also need to resolve this https://github.com/SpeciesFileGroup/taxonworks/issues/1762 we think to give you truly what you want.

There is an opportunity to formalize this into a more controlled checklist, and to think about the relationships of this annotation as an attribute of the data it is linked to.  In part, are these statements about biological taxa, nomenclatural issues, etc.

Ping @debpaul

**Comment by proceps** [comment]
Alternative solution is a citation with a tag.
I understand scrutiny was introduced to SF as a result of collaboration with COL. But COL has different definition of scrutiny. So the really question would be how OSF group is using this information.

**Comment by mjy** [comment]
Tags are not citable.

**Comment by proceps** [comment]
Sorry, not tags, I ment citation topics.

**Comment by MMCigliano** [comment]
If Notes could have an Author and a Year that would work for us and would be similar to the “scrutiny” of OSF.

**Comment by mjy** [comment]
By definition you describe exactly a DataAttribute::Internal with Predicate
'Scrutiny' and a Person based Source in the Citation on it.

Notes will not have citations (because we need them to be *on* citations
potentially, and can't create a circular logic).  They are not controlled
enough for what you want/need.

**Comment by MMCigliano** [comment]
“Scrutiny” in OSF has and author and a year, so that you can search/filter by author name. The content of Scrutinies are nomenclatural comments mostly from experts of the taxa involved, but not published yet. Most of the times it is an information between an expert and the curator of the database.  
So, I do not think that it is or fits in DataAttribute.

**Comment by mjy** [comment]
## Semantics

Whenever we want to say that someone said some thing in TaxonWorks we assert that by using a Citation.  This is a case where someone said something at a time, therefor it is a Source (Person, Year) used in a Citation.  This model gives you the functionality, once built in the interface, to show all "Scrutinies" per person per year.   

What the Citation is on is the next question.

Since you want somethign more free-text your options is a Data Attribute that also refrences a controlled vocabularly term.  The only real semantics issue is that we could add a 'is_metadata' flag on the DataAttribute that asserts that the attribute is not about the thing itself, but about the data on the thing.  We do the same with Depictions already, so their is precident/

You are of course not seeing the filter (user interface) built yet, but the model is fairly clear I think.

Here is the model:

![Screen Shot 2021-03-01 at 9 28 04 AM](https://user-images.githubusercontent.com/22600/109519362-c847b300-7a70-11eb-8e8c-fbd0bc852f0b.png)

## Interface

The standard DataAttibutes interfaces can be used throughout.  We can adopt them to more or less look precisely like your form in a Citable version.

**Comment by proceps** [comment]
The way I understand it, the topic "Scrutiny" on a particular citation, what is really needed. Or a combination of topic and DataAttribute.

**Comment by mjy** [comment]
Citation Topic won't work because there is no free-text value.  ContentTopic would work fine, with Citation on Content, however those only work for notes on OTUs.  From what I read OSF wants to annotate both nomenclature and concepts, but maybe they would be happy with just OTUs.

**Comment by MMCigliano** [comment]
Yes, it will work. But how do I cite people in Content Editor?

![image](https://user-images.githubusercontent.com/33158036/109527378-56806100-7a92-11eb-92bc-b449ac6b0611.png)

**Comment by mjy** [comment]
@jlpereira Bug, Citation button is not unlocking.
![Screen Shot 2021-03-01 at 10 37 39 AM](https://user-images.githubusercontent.com/22600/109528399-3cd31f80-7a7a-11eb-9ec0-c2ef1cfc7a64.png)

**Comment by MMCigliano** [comment]
@mjy, Yes with just OTUs it will work. 
However, I still do not know how to add the year. Additionally, the author is not shown in the citation

![image](https://user-images.githubusercontent.com/33158036/109530574-a01e7b00-7a95-11eb-9e9b-f054ce8afd0f.png)

**Comment by mjy** [comment]
@MMCigliano good catch, that should just be a rendering issue, i.e. the data should be there.
